docker-compose.yml

```diff
@@ -26,6 +26,9 @@ services:
       - mongodb:/data/db
     networks:
       - app-tier
+      - default
```

Comment:
Every container has a network, that's just how Docker works. In this case I'm exposing the Mongo container to the `default` network because only the `default` network is accessible from `localhost`. We still need the `app-tier` network because the RabbitMQ container needs a network with a fixed name to configure the service, and the `default` network (from the container's perspective) is given a random name at startup.